haystack/utils.py

```diff
@@ -79,3 +81,80 @@ def convert_labels_to_squad(labels_file: str):
 
     with open("labels_in_squad_format.json", "w+") as outfile:
         json.dump(labels_in_squad_format, outfile)
+
+def find_answer_start(answer: str, text: str):
+    """
+    get index of beginning of answer in text
```

Comment:
For docstrings: Please start with uppercase and have one empty line between the description and the `:param` section

Reply:
ok